guarddog/analyzer/metadata/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 
 def get_metadata_detectors(ecosystem: ECOSYSTEM) -> dict[str, Detector]:
-    match(ecosystem):
+    match (ecosystem):
```

Comment:
Is this lint enforced?

Reply:
I reformatted everything with Intellij to avoid flake8 screaming, unsure but Intellij did it yes